Editors/README.md

```diff
@@ -22,56 +22,34 @@ The [Swift for Visual Studio Code extension](https://marketplace.visualstudio.co
 
 ## Sublime Text
 
-Before using SourceKit-LSP with Sublime Text, you will need to install the LSP package from Package Control. To configure SourceKit-LSP, open the LSP package's settings. The following snippet should be enough to get started with Swift.
+Before using SourceKit-LSP with Sublime Text, you will need to install the LSP and LSP-SourceKit packages from Package Control. And you're ready to go if you have `xcrun` in the `$PATH`.
 
-You will need the path to the `sourcekit-lsp` executable for the "command" section.
+To configure SourceKit-LSP additionally, open the SourceKit-LSP package's settings by typing in command palette `Preferences: LSP-SourceKit Settings`. The following snippet should be enough to get started with Swift and Objective-C/C++ and the custom path to `sourcekit-lsp` executable.
+
+To make your journey even more pleasant it's recomended to install [Swift-Next](https://github.com/Swift-Next/Swift-Next) package that provides advanced swift syntax highlighting support.
```

Comment:
I believe we have this talk before, when Swift-Next got included in Package Control [initially](https://github.com/wbond/package_control_channel/pull/8843#issuecomment-1814248902). So after all folks decided to not replace any existing ST swift syntax package with this one, but to add tests and add it into the default [packages list](https://github.com/sublimehq/Packages) instead. In short the main reason was that this package is ST4 compatible only, which is not the case for the "default" [Swift](https://github.com/quiqueg/Swift-Sublime-Package) package provides ST3 and ST2 support as well. So here we are. The only thing I can to this: someday I'll complete the tests task, but the day is not today.

The same reasoning is true when we're speaking about the replacement of [Decent Swift Syntax](https://packagecontrol.io/packages/Decent%20Swift%20Syntax) with [Swift-Next](https://github.com/Swift-Next/Swift-Next) in LSP-SourceKit package readme. If it's acceptable to suggest the ST4 only option as the default option for this package — i'm up for it, and would love to update its readme as well. But if you prefer to keep the one that provides better backward capability, so there's nothing to do with LSP-SourceKit's readme.

Reply:
Yes, ST3 is not relevant anymore.
And also `LSP-SourceKit` is ST4-only so if "Swift-Next" is better, it should just recommend that.

But what I'm mainly tried to say is that from the perspective of someone who just reads intructions here, there is not even a mention of installing any syntax so this is why I thought the wording here is not ideal. Many users will likely not read `LSP-SourceKit` readme separately if they are installing it based on those instructions.

Reply:
Yeah, I see your point, but personally I think that it's unlikely that a user would bother to install an lsp plugin for language while intentionally skipping its syntax highlighting part. So this is why I chose this form rather than more strict one.

Reply:
I agree that we should make the suggestion for the syntax highlighting package consistent between the documentation of _LSP-SourceKit_ and this REAMDE. I take it that there is consensus that _Swift Next_ is superior to _Decent Swift Syntax_ (especially because there is a plan to get _Swift Next_ into the default packages) except for the part that _Swift Next_ is not ST4-compatible. But if _LSP-SourceKit_ isn’t either, I don’t think that’s a blocker. Could we

- Change _LSP-SourceKit_ to also refer to _Swift Next_?
- I agree that we should rephrase this sentence to be a little less optional, on the lines of: Also install _Swift Next_ to use Swift in ST.
- I am not familiar with ST’s package infrastructure. But would it be possible to make _LSP-SourceKit_ depend on _Swift Next_ so users get the entire bundle when installing _LSP-SourceKit_?

Reply:
> Change LSP-SourceKit to also refer to Swift Next?

It's updated now - https://github.com/sublimelsp/LSP-SourceKit/pull/6

> I am not familiar with ST’s package infrastructure. But would it be possible to make LSP-SourceKit depend on Swift Next so users get the entire bundle when installing LSP-SourceKit?

It's not technically possible right now for one package to depend on another package. There is a concept of dependencies but those are not packages and can't be installed on its own.

Reply:
> It's updated now - [sublimelsp/LSP-SourceKit#6](https://github.com/sublimelsp/LSP-SourceKit/pull/6)

Thanks!

> It's not technically possible right now for one package to depend on another package. There is a concept of dependencies but those are not packages and can't be installed on its own.

Ah, OK, I wasn’t aware of that. Let’s keep the suggestion in here then.